linux/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -153,6 +153,17 @@
         - unattend_install_conf is defined
         - ('UOS' in unattend_install_conf)
 
+    # For ubuntu desktop, it's invalid with "echo '{{ autoinstall_complete_msg }}'' > /target/dev/ttyS0"
```

Comment:
This comment causes confusing, can be improved

Reply:
Thanks. Done